packages/sync-engine/src/database/migrations/0031_add_default_price.sql

```diff
@@ -0,0 +1,2 @@
+alter table "stripe"."products"
+add column "default_price" text;
```

Comment:
```suggestion
add column IF NOT EXISTS "default_price" text;
```

Just to make migrations a little safer in case anyone added this manually on their end